pkgs/by-name/ba/badgemagic-rs/package.nix

```
@@ -0,0 +1,46 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, dbus
+, udev
+, stdenv
+, darwin
+,
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "badgemagic-rs";
+  version = "0-unstable-2024-08-01";
+
+  src = fetchFromGitHub {
+    owner = "fossasia";
+    repo = "badgemagic-rs";
+    rev = "e7989e7";
+    hash = "sha256-LNcYYMoavF4Ir+LBBx5iU5PviMIoT4szZT0A9PViGtk=";
+  };
+
+  cargoHash = "sha256-T11DNcxg3oPLYGT6s6XmO/py0J8ATMfYEphgVsx3RK8=";
```

```suggestion
  useFetchCargoVendor = true;
  cargoHash = "";
```
We recently switched to a new fetcher for cargo dependencies